runtime/src/nfts.rs

```diff
@@ -39,15 +40,17 @@ decl_module! {
 
             // get the bundled hash
             let bundled_hash = Self::get_bundled_hash(pfs, deposit_address);
-
             Self::deposit_event(RawEvent::DepositAsset(bundled_hash));
 
+			let metadata = bundled_hash.as_ref().to_vec();
+			let resource_id = <T as pallet_bridge::Trait>::HashId::get();
+			<chainbridge::Module<T>>::transfer_generic(dest_id, resource_id, metadata)?;
             Ok(())
```

Comment:
Lets remove the `Ok()` here, since `<chainbridge::Module<T>>::transfer_generic(dest_id, resource_id, metadata)?;` already returns that.

Reply:
This is a standard practice, and makes the code more extensible, such as: whenever you need to add more statement calls before the Ok(()), it doesn't need to be a function that returns a `Result`.

Reply:
I am fine by following the most standard practice, then lets follow the same thing everywhere, like: https://github.com/centrifuge/centrifuge-chain/pull/228/files#diff-ca12cf69a0ed4856af5c862837c21d31R48